### DISCUSSION TITLE
How to find out which flows are enabled?

### DISCUSSION THREAD

**Discussion by NickUfer** [ory/kratos#3379] [Q&A]
Hey,
I implemented a frontend for kratos where users can sign in and register. On the log in page I show a "Don't have an account? Register here" message which links to the registration page.

Kratos has the option to disable registration which was added [here](https://github.com/ory/kratos/pull/2081).
I want to use this feature and automatically remove the link to the registration page on my signin page when registration is disabled.

AFAIK, no kratos API endpoint exposes information on whether a flow is enabled or not. So the best idea I got right now is to create a registration flow on every signin page call and check for errors to find out if registration is enabled or not. 

Is there a better way to find out if a flow is enabled?

**Comment by vinckr** [comment]
Hey Nick, 
So you have one boilerplate frontend and multiple Ory Kratos and in some you disable the registration and in some you dont. And because you use the same frontend for all you want to check what is enabled etc.? Does that describe your setup?
Another approach could be to manage this information on your application side.
Maybe blocking webhooks instead of disabling the flow, but not sure if that helps with the original problem.
